src/util.h

```diff
@@ -170,7 +170,10 @@ static SECP256K1_INLINE void *checked_malloc(const secp256k1_callback* cb, size_
 #define ALIGNMENT 16
 #endif
 
-#define ROUND_TO_ALIGN(size) ((((size) + ALIGNMENT - 1) / ALIGNMENT) * ALIGNMENT)
+/* ceil(x/y) for integers x >= 0 and y > 0. Here, / denotes rational division. */
+#define CEIL_DIV(x, y) (1 + ((x) - 1) / (y))
```

Comment:
This is clearly better than my suggestion, but this can now underflow for unsigned x == 0, so the docs need to be changed:
```suggestion
/* ceil(x/y) for integers x > 0 and y > 0. Here, / denotes rational division. */
#define CEIL_DIV(x, y) (1 + ((x) - 1) / (y))
```

It's a bit sad that there's no real way to add a VERIFY_CHECK in a macro (or at least not one which is compatible with the way we use this macro in `WNAF_SIZE_BITS(bits, w)` as a compile time integer constant...). If you think that's better, we could drop that use, and make it a real function `size_t ceil_div(size_t, size_t)` with a VERIFY_CHECK instead, but no strong opinions here.